easybuild/framework/easystack.py

```diff
@@ -80,6 +85,11 @@ def compose_ec_filenames(self):
             })
             ec_filename = '%s-%s.eb' % (sw.name, full_ec_version)
             ec_filenames.append(ec_filename)
+
+        # entries specified via 'easyconfigs' top-level key
+        for ec in self.easyconfigs:
+            ec_filenames.append(ec + '.eb')
```

Comment:
Why did you decide to add the `.eb` extension here? Why not have people write in the YAML file:

```
robot: true
easyconfigs:
  - binutils-2.25-GCCcore-4.9.3.eb
  - foss-2018a.eb
  - R-bundle-Bioconductor-3.11-foss-2020a-R-4.0.0.eb
  - GROMACS-2020.4-foss-2020a-Python-3.8.2.eb
```
That way, it is immediately clear and recognizeable that these are (full) EasyConfig filenames

Reply:
Because it's redundant. The top-level key should make it very clear what these strings are.

I'm happy to allow the use of `.eb` though, and we can guard the `append` with an `not ec.endswith('.eb')`

Reply:
personally I would find it more intuitive to specify the `.eb` extension, even if it's redundant. No reason not to allow both though, guarding the `append`

Reply:
and it makes for easier C&P

Reply:
I took the liberty of pushing that change :)